src/transformers/configuration_utils.py

```diff
@@ -236,6 +236,10 @@ class PretrainedConfig(PushToHubMixin):
 
         use_bfloat16 (`bool`, *optional*, defaults to `False`):
             Whether or not the model should use BFloat16 scalars (only used by some TensorFlow models).
+        tf_legacy_loss (`bool`, *optional*, defaults to `False`):
```

Comment:
Are we sure we want to set the default to `False`? This is breaking no? Also it's a somewhat hard-to-discover silent error in this case no?

Reply:
It's only very slightly breaking - anyone using Keras or a custom model will not notice any change. The existing losses return very strange shapes like vectors of shape `(num_unmasked_tokens,)` that vary in length each iteration, with no mapping from there back to the original tokens. I doubt anyone is using them directly.